components/04-pages/podcast-pages/podcast-overview-page/_podcast-overview-page.scss

```diff
@@ -0,0 +1,88 @@
+.image-header--podcast {
+  padding: 0;
```

Comment:
soll man nicht vielleicht lieber ein `.topic-header` für diese Seite verwenden? Das sieht für mich ähnlich aus. Für mich ist ein bisschen ein Code Smell, wenn man hier in einer Page das `padding` von einer Komponente überschreiben muss, damit man das gewünschte Aussehen hat. Alternativ braucht man vielleicht ein `.image-header--no-padding` (ggf. mit anderer Name) Variante für die `.image-header` Komponente, aber ich glaube dieses Styling gehört eher dorthin.

Reply:
ja, stimme dir zu!